src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.targets

```diff
@@ -50,6 +50,10 @@ Copyright (c) .NET Foundation. All rights reserved.
     <GenerateRuntimeConfigurationFilesInputs Include="$(UserRuntimeConfig)" Condition=" Exists($(UserRuntimeConfig)) " />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(TargetFrameworks)' != '' And '$(TargetFramework)' != ''">
```

Comment:
When is both TargetFrameworks defined and TargetFramework defined? I thought it was one or the other?

Reply:
In inner builds of multitargeted projects. They're multitargeted because `TargetFrameworks` has value; they're inner because `TargetFramework` does.